manager/dispatcher/discord.go

```diff
@@ -3,6 +3,7 @@ package dispatcher
 import (
 	"bytes"
 	"encoding/json"
+	"errors"
```

Comment:
It seems not related all changes in `manager` with this PR and issue

Reply:
Agree, I removed all the changes that's been committed with PR https://github.com/dsrvlabs/vatz/pull/514.
Can you check again?